docs/docs/references/configuration/cli/trivy_aws.md

```diff
@@ -76,6 +76,8 @@ trivy aws [flags]
       --endpoint string                   AWS Endpoint override
       --exit-code int                     specify exit code when any security issues are found
   -f, --format string                     format (table,json,template,sarif,cyclonedx,spdx,spdx-json,github,cosign-vuln) (default "table")
+      --helm-api-versions strings         Available API versions used for Capabilities.APIVersions. This flag is the same as the api-versions flag of the helm template command. (can specify multiple or separate values with commas: policy/v1/PodDisruptionBudget,apps/v1/Deployment)
+      --helm-kube-version string          Kubernetes version used for Capabilities.KubeVersion. This flag is the same as the kube-version flag of the helm template command.
```

Comment:
Do we need these flags in Trivy AWS?

Reply:
I just run `mage docs:generate` and the other helm parameter are present, too.

I guess its something for a different PR to fix that.

Reply:
OK fair enough. We should remove them from Trivy AWS as I don't think they are needed, but in a different PR is fine.